content/ArtificialIntelligence.txt

```diff
@@ -314,35 +318,57 @@ Sampling size VS uncertainty
 
  - Probability VS statistics
 
-Terms in machine learning
+What is a training dataset?
 
-  Training dataset: a collection of data used to train a machine learning model.
+  A training dataset is a collection of data used to train a machine learning model.
 
-  Example: a data point in the training dataset.
+What is a testing dataset?
 
-  Label: the thing we are trying to predict using a machine learning model.
+  A testing dataset is a collection of data used to evaluate the performance of a trained machine learning model during training. It contains data that the model has not seen during training to evaluate how well the model generalizes to unseen data.
 
-  Features: variables that the predictions are based on.
+What is a validation dataset?
 
-  Testing dataset: a collection of data used to evaluate the performance of a trained machine learning model during training. It contains data that the model has not seen during training to evaluate how well the model generalizes to unseen data.
+  A validation dataset is a collection of data used to evaluate the performance of a final machine learning model. The model should not have seen this data (or similar data) during training and testing!
 
-  Validation dataset: a collection of data used to evaluate the performance of a final machine learning model. The model should not have seen this data (or similar data) during training and testing!
+What is a label in machine learning?
 
-  Weights: the importance of each input feature in predicting the output.
+  A label is a classification of a datapoint.
```

Comment:
We want the model to learn the relationship between datapoints and their labels to predict labels for new datapoints.